content/mailing_lists.html

```diff
@@ -184,6 +184,13 @@ <h2 id="security">Security</h2>
 <p>In order to post to the list, it is <strong>NOT</strong> necessary to first subscribe
 to it.</p>
 <p>For information on published vulnerabilities please see our <a href="cve_list.html">CVE list</a>.</p>
+<p>This mailing list is only for discussing security vulnerabilities in hadoop &lsquo;source&rsquo; code, <strong>NOT</strong> security vulnerabilities coming via thirdparty libraries. For security issues related to thirdparty libraries use the dev/user mailing lists.</p>
+<p>The thridparty library versions in the upcoming releases can be checked here:</p>
```

Comment:
Spotted a small typo:

```suggestion
<p>The thirdparty library versions in the upcoming releases can be checked here:</p>
```